src/App.tsx

```diff
@@ -1,3 +0,0 @@
-export function App() {
-  return <div className='App'></div>;
-}
```

Comment:
`App.tsx` shouldn't be removed. We should keep this because this is our root component.

We should use `react-router` to define routing path and display appropriate pages for given path:

https://v5.reactrouter.com/web/example/route-config

Reply:
See how to use react-router with mui:

https://stackoverflow.com/a/63221012/9095988